Gemfile

```diff
@@ -54,7 +54,7 @@ end
 group :development do
   gem 'quiet_assets'
   gem 'better_errors'
-  # gem 'binding_of_caller'
+  gem 'binding_of_caller'
```

Comment:
I suppose Andre did that for some debugging. @carpodaster : I enabled it as well, in a local debugging branch. Do you mean we shouldn't introduce it at all, or just not as a part of THIS particular PR?